app/routes/__index/admin/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { Container, Flex, Heading, Input, InputGroup, InputLeftAddon } from '@chakra-ui/react';
+import { FaUsers, FaSearch, FaStickyNote } from 'react-icons/fa';
+import { TbFileCertificate } from 'react-icons/tb';
+import AdminMetricCard from '~/components/admin/admin-metric-card';
+import UsersTable from '~/components/admin/users-table';
+import mockUsers from '~/mock/users';
+
+export default function AdminRoute() {
+  return (
+    <Container maxW="container.xl">
+      <Heading as="h1" size="xl" mt="8">
+        Admin panel
+      </Heading>
+      <Flex flexWrap="wrap">
+        <AdminMetricCard
+          name="Users"
+          tooltipText="Total number of users"
+          value="1923"
+          IconComponent={FaUsers}
+        />
+        <AdminMetricCard
+          name="Certificates"
+          tooltipText="Total number of certificate"
+          value="1532"
+          IconComponent={TbFileCertificate}
+        />
+        <AdminMetricCard
+          name="Records"
+          tooltipText="Total number of Records"
+          value="10394"
+          IconComponent={FaStickyNote}
+        />
+      </Flex>
+      <Heading as="h2" size="xl" mt="8" mb="4">
+        Users
+      </Heading>
+      <InputGroup width={{ sm: '100%', md: 300 }}>
+        <InputLeftAddon children={<FaSearch />} />
+        <Input placeholder="Search..." />
```

Comment:
I was just wondering... are you perhaps waiting for a code to make the search functional?

Reply:
This could probably be done in its own issue, not sure if we have one filed

Reply:
Let's get this filed for sure.